README.md

```diff
@@ -25,3 +26,16 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+basecase is 0 if t(n) is less than or equal to 2
+
+T(n) = 3 * T(n/3) + O(n)
```

Comment:
What is $O(n)$ here? You can't add a complexity class to an expression like this.